hydroflow_cli_integration/src/lib.rs

```diff
@@ -71,10 +74,20 @@ impl ServerBindConfig {
             ServerBindConfig::TcpPort(host) => {
                 let listener = TcpListener::bind((host, 0)).await.unwrap();
                 let addr = listener.local_addr().unwrap();
-                BoundConnection::TcpPort(
-                    tokio::spawn(async move { Ok(listener.accept().await?.0) }),
-                    addr,
-                )
+                let (conn_send, conn_recv) = tokio::sync::mpsc::unbounded_channel();
+
+                tokio::spawn(async move {
+                    loop {
+                        if conn_send
+                            .send(listener.accept().await.map(|r| r.0))
+                            .is_err()
+                        {
+                            break;
+                        }
+                    }
+                });
```

Comment:
Maybe makes sense to use https://docs.rs/tokio-stream/0.1.14/tokio_stream/wrappers/struct.TcpListenerStream.html which is `Stream<Item = Result<TcpStream>>` wrapper around the `listener`

Instead of the separate task and mpsc unbounded

Reply:
#659